storage/testsuite/test_parallel.go

```diff
@@ -22,7 +22,8 @@ func testParallel(t *testing.T, store storage.KeyValueStore) {
 	defer cleanupItems(store, items)
 
 	for i := range items {
-		item := items[i]
+		index := i
```

Comment:
This is needed, there was a bug, however the test didn't fail because the index is only used for a test message output.